src/styles/card.scss

```diff
@@ -0,0 +1,136 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $card-width;
+
+  // height: 408px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: $color-white;
+  justify-content: center;
+
+  &__image {
+    display: block;
+
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    display: block;
+    height: 36px;
+    font-family: Roboto, sans-serif;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $card-main-color;
+    box-sizing: border-box;
+    margin: 0 0 4px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    transition: transform 300ms ease-in-out;
+  }
+
+  &__name:hover {
+    color: $color-hover;
+    transition: color 300ms ease-in-out;
+  }
+
+  &__code {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    color: $card-secondary-color;
+    box-sizing: border-box;
+    margin-bottom: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: center;
+    margin-top: 5px;
+    line-height: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    box-sizing: border-box;
+  }
+
+  &__reviews {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    color: $card-main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    box-sizing: border-box;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $card-secondary-color;
+  }
+
+  &__price-value {
+    font-family: Roboto, sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $card-main-color;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    box-sizing: border-box;
+    height: 40px;
+    border: 1px solid $card-buy-color;
+    background-color: $card-buy-color;
+
+    text-decoration: none;
+    border-radius: 5px;
+    font-family: Roboto, sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    color: $color-white;
+    text-transform: uppercase;
+
+    &:hover {
+      border: 1px solid $card-buy-color;
+      background-color: #fff;
```

Comment:
The hover effect changes the background color to white, which might not be visible if the card's background is also white. Ensure there's enough contrast for the hover state to be noticeable.